MCprep_addon/materials/skin.py

```diff
@@ -703,6 +733,77 @@ def execute(self, context):
 		return {'FINISHED'}
 
 
+class MCPREP_OT_swap_skin_variant(bpy.types.Operator, spawn_util.VariationProp):
+	"""Apply the active UIlist skin to select characters"""
+	bl_idname = "mcprep.swap_skin"
+	bl_label = "Swap mob skin"
+	bl_description = "Swap the mobs variant"
```

Comment:
Same comment about i18n here too
```suggestion
	bl_label = env._("Swap mob skin")
	bl_description = env._("Swap the mobs variant")
```